tools/traffic/workers/blob_writer.go

```diff
@@ -0,0 +1,164 @@
+package workers
+
+import (
+	"context"
+	"crypto/md5"
+	"crypto/rand"
+	"fmt"
+	"github.com/Layr-Labs/eigenda/api/clients"
+	"github.com/Layr-Labs/eigenda/encoding/utils/codec"
+	config2 "github.com/Layr-Labs/eigenda/tools/traffic/config"
+	"github.com/Layr-Labs/eigenda/tools/traffic/metrics"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"sync"
+	"time"
+)
+
+// BlobWriter sends blobs to a disperser at a configured rate.
+type BlobWriter struct {
+	// The context for the generator. All work should cease when this context is cancelled.
+	ctx *context.Context
+
+	// Tracks the number of active goroutines within the generator.
+	waitGroup *sync.WaitGroup
+
+	// All logs should be written using this logger.
+	logger logging.Logger
+
+	// Config contains the configuration for the generator.
+	config *config2.WorkerConfig
+
+	// disperser is the client used to send blobs to the disperser.
+	disperser clients.DisperserClient
+
+	// Unconfirmed keys are sent here.
+	unconfirmedKeyHandler KeyHandler
```

Comment:
have you considered passing unconfirmed keys to a channel instead and letting the caller handle it? 
I think that's a common pattern in Go

Reply:
I'm willing to make this change, but since this modifies the API, would it be ok with you if I made this change in the next PR that will add the code that consumes the unconfirmed keys? It will cause some merge conflicts if I change this API in this PR.